.travis.yml

```diff
@@ -1,8 +1,7 @@
 language: node_js
 node_js:
-  - 4
   - 6
   - 8
-  - 9
+  - 10
```

Comment:
Are you removing Node 9 since it's not an LTS release?

Reply:
yeah, its not supported anymore. I#d add 11 if you want to?

Reply:
Nah it's fine.